.github/workflows/build.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     - uses: actions/checkout@v1
       with:
         submodules: recursive
-    - uses: eine/setup-msys2@v1
+    - uses: eine/setup-msys2@v2
```

Comment:
This was already implemented with #6982

Reply:
Oh cool. So it was merged in the meantime.

Reply:
Reverted and rebased.